libs/features/components/banners/src/lib/components/elewa-group-team-member/elewa-group-team-member.component.html

```diff
@@ -0,0 +1 @@
+<p>elewa-group-team-member works!</p>
```

Comment:
There's not template code on this file??

Reply:
Hi, I did a second PR, which has the most updated code: https://github.com/italanta/elewa-group/pull/132